### PR TITLE
[CXF-9046] Close response objects in perf client sample

### PR DESCRIPTION
**Pull request by jgoodyear**

### CODE REVIEW

File: distribution/src/main/release/samples/performance/jaxrs/src/main/java/org/apache/cxf/performance/https/client/Client.java [hunk 1]
```diff
@@ -85,13 +85,15 @@ public void doJob(WebClient webClient) {
...
                     Asserts.check(respGet.getStatus() == 200, "Get should have been OK");
+                    respGet.close();
                     break;
                 case 1:
                     //POST
                     Customer customerPost = new Customer();
                     customerPost.setName("Jack");
                     try (Response respPost = webClient.post(customerPost)) {
                         Asserts.check(respPost.getStatus() == 200, "Post should have been OK");
+                        respPost.close();
```
Comment (reta) [inline distribution/src/main/release/samples/performance/jaxrs/src/main/java/org/apache/cxf/performance/https/client/Client.java:96]: @jgoodyear you don't need that, `try (Response respPost = webClient.post(customerPost)) {` closes the response (it is `AutoCloseable`)

Comment (jgoodyear) [reply distribution/src/main/release/samples/performance/jaxrs/src/main/java/org/apache/cxf/performance/https/client/Client.java:96]: I'm happy to remove from the POST call - the GET call certainly has lingering connections

Comment (reta) [reply distribution/src/main/release/samples/performance/jaxrs/src/main/java/org/apache/cxf/performance/https/client/Client.java:96]:
```
try (Response respGet = webClient.get()) {
     Asserts.check(respGet.getStatus() == 200, "Get should have been OK");
}
```

Comment (jgoodyear) [reply distribution/src/main/release/samples/performance/jaxrs/src/main/java/org/apache/cxf/performance/https/client/Client.java]: I'll give it a try and reply here.

**Review by reta — APPROVED** [review]

**Pull request merged**